fuels-abigen-macro/README.md

```diff
@@ -40,7 +40,8 @@ fn compile_bindings_from_inline_contract() {
         "#
     );
 
-    let contract_instance = SimpleContract::new();
+    let dummy_client = FuelClient::new("").unwrap();
```

Comment:
The string here is supposed to be the URL? Instead of a string with a magic value, could we use an `Option`?

Reply:
Yeah, `""` is just too... ugly. Passing a `None` would make more sense, even though very few times a user will pass a `None` (or `""`) to this method.

Reply:
Changing this can be left to a future PR, since it's a functional change.

Reply:
@vnepveu can you file an issue?

Reply:
Hmm the problem is that this is on the `fuel-core` side https://github.com/FuelLabs/fuel-core/blob/master/fuel-client/src/client.rs#L52.

Reply:
Ah, then we can file an issue in `fuel-core`!

Reply:
Yes filing the issue. Should I update the `""` bit to a random non-empty string?

Reply:
Well, no, since it's a URL. If the correct thing _currently_ is `""`, then the docs should say `""`.